i18n.install

```diff
@@ -50,13 +64,162 @@ function i18n_update_last_removed() {
 }
 
 /**
- * Move book settings from variables to config.
+ * Empty hook.
  */
 function i18n_update_1000() {
-  // Migrate variables to config.
-  // $config = config('i18n.settings');
-  // $config->set('book_allowed_types', update_variable_get('book_allowed_types', array('book')));
-  // $config->save();
-  // Delete variables.
-  // update_variable_del('book_allowed_types');.
+  // Got removed.
+}
+
+/**
+ * Disable unavailable and deprecated modules.
+ */
+function i18n_update_1001() {
+  // Neither can we, nor do we need to run any code, just rip out of the system
```

Comment:
I know what is happening here but this comment doesn't make sense to me. Perhaps "We don't want to run any code from these modules so just remove them the system table.

Reply:
Sure, the comment could be clearer. Fact is, that we can not run any code - these modules got removed, there's no code anymore.